addons/hr_timesheet/migrations/10.0.1.0/post-migration.py

```diff
@@ -5,6 +5,21 @@
 from openupgradelib import openupgrade
 
 
+def migrate_use_timesheets(env):
+    openupgrade.logged_query(
```

Comment:
You shouldn't change original `use_timesheets` value, as this will be incorrectly transferred later to projects. See for example this case:

- You have an analytic account (AA) using timesheets.
- You input some timesheets.
- But later that AA is deprecated and you mark it for not entering timesheets anymore.
- When migrating, you'll have the project linked to the AA with the mark "Use timesheets" again.